src/rails_admin/styles/base/variables.scss

```diff
@@ -1 +1,3 @@
 /* RailsAdmin SASS variables */
+
+$font-size-base: 0.85rem !default;
```

Comment:
Bootstrap5's default font size is slightly larger than Bootstrap3, so changing it smaller